### ISSUE TITLE
Marco Checkers: Irritating `if you can jump, you must!`-message

### ISSUE THREAD

**Issue by tolik518** [evanbowman/skyland-beta#14]
In the `macro_checkers`-minigame if you want to move a token (playing piece) when another is able to jump
it will prompt you with the message `if you can jump, you must!`.
For new players, it is pretty irritating what it means. I had no idea for instance what it means since I tried to move my token.

Maybe you could flash the token that should move so it gets less ambiguous and the player knows what he has to do.

**Comment by evanbowman** [comment]
I'll see if I can fix it! Forced jumps are a common rule in the version of checkers played where I live.

**Comment by tolik518** [comment]
> I'll see if I can fix it! Forced jumps are a common rule in the version of checkers played where I live.

Yeah I think it's a good rule, but it's a bit irritating if you dint know the game properly 😅
